bodhi/models/models.py

```diff
@@ -1649,19 +1649,35 @@ def check_requirements(self, session, settings):
 
         return True, "All checks pass."
 
+    def negative_karma(self):
+        """Checks for negative karma, Returns True if the update has negative karma"""
```

Comment:
Consider renaming the method to sound a little bit more actionable. Something like ```has_negative_karma()``` is a little clearer, as it hints that it might be returning a boolean value. This is just a nitpick, so you can leave it as is if you prefer.